src/main/java/org/folio/bulkops/processor/UpdatedEntityHolder.java

```diff
@@ -10,4 +10,5 @@
 public class UpdatedEntityHolder<T extends BulkOperationsEntity> {
   private T updated;
   boolean shouldBeUpdated;
+  private int errors;
```

Comment:
For which purposes we need to save errors counter here if all this information should be available in the bulk operation entity?

Reply:
This holder should be used for only 2 purposes: 1) contains EXPECTED update; 2) determine if entity should be updated or no based on the fact there are validation issues or entity doesn't contain any changes. Please, avoid to extend it with unlogical data.